tests/test_cookie_helpers.py

```diff
@@ -1137,6 +1137,35 @@ def test_parse_cookie_header_empty() -> None:
     assert parse_cookie_header("   ") == []
 
 
+# https://github.com/aio-libs/aiohttp/issues/11632
+@pytest.mark.xfail
+def test_parse_cookie_gstate_header() -> None:
+    header = (
+        '_ga=ga; '
+        'ajs_anonymous_id=0anonymous; '
+        'analytics_session_id=session; '
+        'cookies-analytics=true; '
+        'cookies-functional=true; '
+        'cookies-marketing=true; '
+        'cookies-preferences=true; '
+        'g_state={"i_l":0,"i_ll":12345,"i_b":"blah"}; '
+        'analytics_session_id.last_access=1760128947692; '
+        'landingPageURLRaw=landingPageURLRaw; '
+        'landingPageURL=landingPageURL; '
+        'referrerPageURLRaw=; '
+        'referrerPageURL=; '
+        'formURLRaw=formURLRaw; '
+        'formURL=formURL; '
+        'fbnAuthExpressCheckout=fbnAuthExpressCheckout; '
+        'is_express_checkout=1; '
```

Comment:
Black is trying to reformat everything with `"`. I'm not clear why pre-commit is not pushing a formatting commit automatically though...

Reply:
hmm, I think i forgot to install pre-commit, re-ran with it enabled